coalib/settings/Setting.py

```diff
@@ -13,6 +14,22 @@ def path_list(obj, *args, **kwargs):
     return obj.__path_list__(*args, **kwargs)
 
 
+def glob(obj, *args, **kwargs):
+    """
+    Creates a path in which all special glob characters in all the
+    parent directories in the given setting are properly escaped.
+    """
+    return obj.__glob__(*args, **kwargs)
+
+
+def glob_list(obj, *args, **kwargs):
+    """
+    Creates a list of paths in which all special glob characters in all the
+    parent directories of all paths in the given setting are properly escaped.
+    """
+    return obj.__glob_list__(*args, **kwargs)
+
+
```

Comment:
@Makman2 it makes sense to have it here though ... because we want bear users to use it in bears and it is a setting type.